Makefile

```diff
@@ -35,13 +35,11 @@ bin/kubectl-kuttl_$(KUTTL_VERSION):
 bin/kubectl-kuttl: bin/kubectl-kuttl_$(KUTTL_VERSION)
 	ln -sf ./kubectl-kuttl_$(KUTTL_VERSION) bin/kubectl-kuttl
 
-.PHONY: install-bin
-install-bin: bin/kind bin/kubectl-kuttl
 
 .PHONY: create-kind-cluster
 create-kind-cluster: $(KUBEADDONS_TEST_KUBECONFIG)
 
-$(KUBEADDONS_TEST_KUBECONFIG): install-bin
+$(KUBEADDONS_TEST_KUBECONFIG): bin/kind bin/kubectl-kuttl
```

Comment:
Let's do these Makefile changes in a different PR so it's clearer that this is specifically a new test.

Reply:
touched on via #13